build_tools/.gitignore

```diff
@@ -0,0 +1 @@
+repo
```

Comment:
This could also download into https://github.com/nod-ai/TheRock/tree/main/third-party or some other location, like the somewhere under the user's home dir. WDYT?

(Moot point if we switch from repo to git submodules)

Reply:
I don't love this but can't think of a strictly better way to do it now. Let's do this and then revisit as we decide what to do ala repo in the longer term.